src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -307,7 +307,7 @@ public List<RESTInformatieobjecttype> listInformatieobjecttypesForZaak(@PathPara
     @POST
     @Path("informatieobject/upload/{uuid}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response uploadFile(@PathParam("uuid") final UUID uuid, @MultipartForm final RESTFileUpload data) {
+    public Response uploadFile(@PathParam("uuid") final String uuid, @MultipartForm final RESTFileUpload data) {
```

Comment:
Waarom doe je dat? Die parameter kan tocg gewoon een UUID zijn? ... net als bij de andere methods.

Reply:
Vreemd - ik kreeg een server error met iets over concatenation in de beschrijving die uit regel 311 ontstond bij het uploaden van een file. Die exception kan ik nu niet meer reproduceren dus ik weet ook niet zeker wat daar nou de oorzaak van was. Dit lijkt inderdaad overbodig, dus ik haal de oude situatie weer terug.

Reply:
Besproken met Marco en tot een oplossing gekomen in opvolgende commit - toch weer String van gemaakt. Reden waarom dit nodig is beschreven in toegevoegde javadoc. Ook usages in beide front- en back-end gecontroleerd om zeker te zijn dat geen verdere refactor nodig is.